src/styles/sidebar/components/login-control.scss

```diff
@@ -1,27 +1,8 @@
-.login-control{
+.login-control {
   flex-shrink: 0;
 }
 
 .login-text {
   font-size: $body2-font-size;
   padding-left: 6px;
 }
-
-/* The user account dropdown menu */
-.login-control-menu {
```

Comment:
These styles were used for the inconsistent single-annotation-view variant of the menu and we've decided to make that consistent with the sidebar rendering, so these styles are not needed anymore.